### DISCUSSION TITLE
Complete `receive-pack` and `upload-pack` support

### DISCUSSION THREAD

**Discussion by xpepermint** [GitoxideLabs/gitoxide#70] [Ideas]
I would be very interested in having the support for Git server operations `receive-pack` and `upload-pack`. I see this mentioned in the README but there's no timeline.

Refs: https://github.com/rust-lang/git2-rs/issues/591, https://github.com/libgit2/libgit2/issues/5605

**Comment by Byron** [comment]
Timelines are hard for open source projects and I don't think there will be one in the next future. However, you are welcome to check [the project boards](https://github.com/Byron/gitoxide/projects) from time to time to see what's likely to be worked on next or currently in progress.

Giving the future telling a shot though I wouldn't be surprised if similar functionality would become available in the course of the year.

**Reply by xpepermint** [reply]
Yeah, I know that very well. I appreciate your efforts here. I've been searching for this feature for a while so I'll be watching project boards as suggested and I hope this gets there soon :).

**Comment by xpepermint** [comment]
Any new moves here?

**Reply by Byron** [reply]
No, except for: It will happen but work on it didn't start yet.
